libs/api/src/main/java/de/cyface/api/ClassifiedSegmentRetriever.java

```diff
@@ -0,0 +1,189 @@
+/*
```

Comment:
This file and multiple ones like it are not required by the Data-Collector. Are they really necessary as part of this API library? I guess mostly the Authentication and Authorization stuff is really shared code. Everything else seems to belong to one of the other projects and should probably be reintegrated there (but I might be wrong I have no total overview at the moment).

Reply:
Right. I can move them to the `data-provider` easily after merging `data-provider` and `data-exporter` where we already have a task: https://cyface.atlassian.net/browse/RFR-24 Will do so now.